omnigibson/systems/macro_particle_system.py

```diff
@@ -850,7 +842,6 @@ def create(cls, name, n_particles_per_group, create_particle_template, min_scale
 
         Args:
             name (str): Name of the visual particles
-            n_particles_per_group (int): Number of particles to generate per group of these particles
```

Comment:
is there a reason why this was removed? are we okay assuming all macro particle systems will have the same number of particles sampled for each type?

Reply:
I removed it because I felt like it should belong to `Covered` state, rather than `MacroParticleSystem`. This is my attempt to unify with the implementation of `MicroParticleSystem` so that `generate_group_particles_on_object` take in `max_samples` as an external argument to decide on the number of particles to sample, rather than using an internal variable like `n_particles_per_group`.